### ISSUE TITLE
Using diffrent port for sending and recieving UDP packets.

### ISSUE THREAD

**Issue by aftnix** [sippy/rtpproxy#35]
The architecture is following : 

client--------------->kamailio+rtpproxy------>softswitch

I'm concerned with the client--->rtpproxy path.

rtpproxy uses the same port from receiving UDP packets from client and sending udp packets to client. Can this be changed, like they will use different ports? From README, it seems the algorithm rtpproxy follows when establishing session, dictates the port usage described above. So i think to achieve what i'm describing i have to change the src of rtpproxy. Which section of the code i should concentrate on? (I'm willing to submit patches, this behaviour then will be toggled from command line)

**Comment by jevonearth** [comment]
Yes, rtpproxy does symmetric RTP, meaning it will use the same port for sending and receiving between it and the client.

To make the behaviour asymmetric, It would need to be patched. 

Can you explain the use case for asymmetric behaviour here?
